src/pages/List.js

```diff
@@ -231,7 +244,26 @@ export default function List({ token }) {
           id="thesearch"
           onChange={(e) => setQuery(e.target.value)}
         />
-        <IconButton onClick={handleReset} icon="X" label="clear input" />
+        <IconButton
+          onClick={handleReset}
+          icon={
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              class="h-6 w-6"
+              fill="none"
+              viewBox="0 0 24 24"
+              stroke="currentColor"
+            >
+              <path
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                stroke-width="2"
```

Comment:
What a MUCH better looking `X` icon than the character.

All that's left is to JSX-ify these properties!